src/components/ui/index.js

```diff
@@ -0,0 +1,3 @@
+export { CoralIconButton } from './CoralIconButton';
+export { CoralIcon } from './CoralIcon';
+export { CoralButton } from './CoralButton';
```

Comment:
I'm running `npm lint` and brings a lot of warnings but not that one. Can we update the .eslint with the correct settings?

That one should be http://eslint.org/docs/rules/eol-last